### ISSUE TITLE
Allow multiple fixtures paths

### ISSUE THREAD

**Issue by soullivaneuh** [hautelook/AliceBundle#404]
Since v2, we can define only one fixtures path on configuration file:

```
hautelook_alice:
    fixtures_path: tests/fixtures/orm
```

It would be great to be able to specify multiples paths as it was possible trough command line on v1:

```
hautelook_alice:doctrine:fixtures:load --purge-with-truncate --fixtures=src/AppBundle/DataFixtures/ORM --fixtures=src/PowerDNSBundle/DataFixtures/ORM
```

**Comment by micayael** [comment]
Hi. Is there an update about this?

**Comment by theofidry** [comment]
So far no one came with a PR attempting to address this so no :)

**Comment by wRLSS** [comment]
Hey, here is a PR targeting this feature. 
https://github.com/hautelook/AliceBundle/pull/467

**Comment by theofidry** [comment]
Closed by #467
